proto/cosmos/group/v1beta1/events.proto

```diff
@@ -20,18 +20,18 @@ message EventUpdateGroup {
   uint64 group_id = 1;
 }
 
-// EventCreateGroupAccount is an event emitted when a group account is created.
-message EventCreateGroupAccount {
+// EventCreateGroupPolicy is an event emitted when a group policy is created.
+message EventCreateGroupPolicy {
 
-  // address is the address of the group account.
+  // address is the account address of the group policy.
```

Comment:
How about renaming account address => policy address here?
```suggestion
  // address is the policy address of the group policy.
```

Reply:
this is just to denote that a group policy has an account address explicitly
your suggestion looks a bit redundant, wdyt?

Reply:
makes sense